index.js

```diff
@@ -12,6 +13,8 @@ const amount = process.env.AMOUNT;
 const denom = process.env.DENOM;
 const minutes = parseInt(process.env.MINUTES);
 const memo = process.env.MEMO;
+const lcdhost = process.env.LCD_HOST;
```

Comment:
I don't see you need these. The `lcdAddress` is already there and you are using that value.